packages/rke2-runtimeclasses/charts/templates/_helpers.tpl

```diff
@@ -0,0 +1,62 @@
+{{/*
```

Comment:
I think we can get rid of this file entirely. We're not generating names, not labeling or selecting anything, and don't use a service account. All we're doing is installing the RuntimeClass definitions which have fixed names and no labels.